app/src/main/java/com/example/plan2meet/dao/ICalendarDAO.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.plan2meet.dao
+
+import com.example.plan2meet.dto.Calendar
+import retrofit2.Call
+
+interface ICalendarDAO {
+    fun getAllCalendars() : Call<ArrayList<Calendar>>
```

Comment:
Is this an actual Retrofit call you plan to make?  If so, it's missing the @GET endpoint.